src/ingestion/models/ingestionManager.ts

```diff
@@ -7,18 +7,38 @@ import { FileNotFoundError, GdalInfoError } from '../errors/ingestionErrors';
 import { SourcesValidationResponse } from '../interfaces';
 import { GpkgError } from '../../serviceClients/database/errors';
 import { LogContext } from '../../utils/logger/logContext';
+import { InfoData } from '../schemas/infoDataSchema';
+import { GdalInfoManager } from './gdalInfoManager';
 
 @injectable()
 export class IngestionManager {
   private readonly logContext: LogContext;
 
-  public constructor(@inject(SERVICES.LOGGER) private readonly logger: Logger, private readonly sourceValidator: SourceValidator) {
+  public constructor(
+    @inject(SERVICES.LOGGER) private readonly logger: Logger,
+    private readonly sourceValidator: SourceValidator,
+    private readonly gdalInfoManager: GdalInfoManager
+  ) {
     this.logContext = {
       fileName: __filename,
       class: IngestionManager.name,
     };
   }
 
+  public async getInfoData(inputFiles: InputFiles): Promise<InfoData[]> {
+    const logCtx: LogContext = { ...this.logContext, function: this.getInfoData.name };
+
+    const { originDirectory, fileNames } = inputFiles;
+    this.logger.info({ msg: 'getting gdal info for files', logContext: logCtx, metadata: { originDirectory, fileNames } });
+
+    await this.sourceValidator.validateFilesExist(originDirectory, fileNames);
+    this.logger.debug({ msg: 'Files exist validation passed', logContext: logCtx, metadata: { originDirectory, fileNames } });
+
+    const filesGdalInfoData = await this.gdalInfoManager.getInfoData(originDirectory, fileNames);
+
+    return filesGdalInfoData;
+  }
+
```

Comment:
validateSources logs - please change all logs from "info" to "debug" except for the first log (stay's info)

Reply:
validateFilesExist method log's should be debug

Reply:
validateInfoData method log's should be debug

Reply:
validateGpkgFiles method log's should be debug

Reply:
done